packages/google_maps_flutter/google_maps_flutter_ios/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 2.13.2
+
+* Fixes map objects (markers, polygons, polylines, circles, and tiles) being added with the wrong properties
```

Comment:
This is missing a period; please see the style guide linked from the PR checklist.

Reply:
Fixed